packages/bundler/src/commands/inline.js

```diff
@@ -50,14 +54,15 @@ async function inlineSassDependencies(
   ];
   const inlinedDependencies = (await Promise.all(
     allPossibleDependencies.map(async dependency => {
-      const [packageFolder, scssFolder] = await findSassModule(dependency, cwd);
-      if (packageFolder) {
+      const modules = await findSassModule(dependency, cwd);
+      if (modules) {
+        const [scssFolder] = modules;
         const dependencyOutputFolder = path.join(vendorFolder, dependency);
 
         await fs.copy(scssFolder, dependencyOutputFolder);
 
         return [dependency, dependencyOutputFolder];
-      }
+      } else return false;
```

Comment:
Quick style-nit, if we return early in the `if` block then we don't need `else` and can just return false in the same block 👍